src/App.tsx

```diff
@@ -1,13 +1,25 @@
-
-
-
+import { Link } from "react-router-dom";
+import axios from "axios";
+import { useEffect, useState } from "react";
 export default function App() {
   
-
+  const [randomURL, setRandomURL] = useState("")
+   useEffect(() => {
+      axios.get("http://localhost:4000/randomid")
```

Comment:
```suggestion
      axios.get(`${SERVER_URL}/api/v1/getslug)
```
We have the new endpoint that was created `/api/v1/getslug`, so we can use that.

For `SERVER_URL`, ill make it so that it auto-links to the production server or `localhost:4000`, which you can import from a file i will create in the root directory

Reply:
SERVER_URL can now be imported from `common/common.ts` after https://github.com/CommunityFocus/cf-frontend/pull/15 gets merged

Reply:
Noted!